tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -120,6 +120,7 @@ if [ $DEVICE_TYPE = "gpu" ]; then
   fi
   if [[ $INSTALL_DEPS_DISTRIBUTED_SETUP = true ]]; then
     source ${0/%install_deps.sh/install_openmpi.sh}
+    ${PYTHON_EXE} -m pip install deepspeed
```

Comment:
@snnn ORTModule tests depends on deepspeed module. What are the palces we should add this dependency?

Reply:
I initially tried to have it installed from within the python test file. But got permission denied errors since the test file was not run with enough privileges.
Not sure what the best process is here. Maybe another flag in the ```run_dockerbuild.sh``` for this dependency. Or a path to all dependent packages to ```run_dockerbuild.sh``` file?

Reply:
Could you put it in https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/scripts/training/requirements.txt ?

Reply:
Spoke to @edgchen1 and he was in agreement with putting it in https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/scripts/training/requirements.txt. Will go ahead and do that.